Whatsapp_Chat_Exporter/__main__.py

```diff
@@ -239,6 +260,13 @@ def main():
         metavar="phone number",
         help="Exclude chats that match the supplied phone number"
     )
+    parser.add_argument(
+        "--dont-filter-empty",
+        dest="filter_empty",
```

Comment:
Something is not clear to be here.
Did you turn this option to be on by default?

The name and the help messages feels the opposite meaning

Reply:
> Did you turn this option to be on by default?

Yes. This filtering out empty chats is on by default. Actually, there was a logic to prevent empty chats get rendered, but it just not really that useful.

Reply:
It's not clear to me why the arg name is `--dont-filter-empty`
But the dest name is `filter_empty` with `action='store_true',`

And again I'm not python developer so I could be missing something very obvious :)

Reply:
I understand your concern. Would d95b075 be clearer?